### PR TITLE
Feat/future fesom26 comp

### PR DESCRIPTION
**Pull request by ackerlar**

### CODE REVIEW

File: configs/components/oasis3mct/oasis3mct.env.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ add_compiletime_environment_changes:
                                 FCFLAGS: '"-fPIC"'
                 levante:
                         add_export_vars:
-                                OASIS_FFLAGS: '"-fPIC"'
+                                OASIS_FFLAGS: '"-march=core-avx2 -mtune=core-avx2 -fPIC"'
```
Comment (mandresm) [inline configs/components/oasis3mct/oasis3mct.env.yaml]: Do you really need this change? As far as I know this was just resulting into a lot of warnings during compilation. @JanStreffing?

**Review by mandresm — COMMENTED** [review]
I only have one comment, see below

File: configs/components/oasis3mct/oasis3mct.env.yaml (hunk 1 above)
Comment (ackerlar) [reply configs/components/oasis3mct/oasis3mct.env.yaml]: You are right, these changes are not needed. I tried to undo them with a later commit

**Comment by mandresm** [comment]
@ackerlar, let me know when it's ready to be approved/merged

**Comment by ackerlar** [comment]
Hi @mandresm the "files changed" looks good to me

**Review by mandresm — APPROVED** [review]

**Comment by mandresm** [comment]
#approve-changes

**Comment by mandresm** [comment]
#bump

**Pull request merged**